core/src/test/java/org/frankframework/http/rest/ApiListenerServletTest.java

```diff
@@ -913,6 +926,33 @@ public void apiListenerWithRepeatableMessageAndGloballyDisabled() throws Excepti
 		assertTrue(result.containsHeader("pragma"));
 	}
 
+	@Test
+	public void apiListenerWithHeadMethodCall() throws Exception {
+		// Arrange
+		String uri = "/apiListenerWithHeadMethodCall";
+		Message repeatableMessage = Message.asMessage(new Message("{\"tralalalallala\":true}").asByteArray());
+		new ApiListenerBuilder(uri, List.of(Methods.HEAD))
+				.withResponseContent(repeatableMessage)
+				.build();
+
+		Map<String, String> headers = new HashMap<>();
+		headers.put("Accept", "application/json");
+		headers.put("content-type", "application/json");
+
+		// Act
+		Response result = service(createRequest(uri, Methods.HEAD, null, headers));
+
+		// Assert
+		assertEquals(200, result.getStatus());
+		assertEquals("OPTIONS, HEAD", result.getHeader("Allow"));
+		assertNull(result.getErrorMessage());
+		assertFalse(result.containsHeader("etag"));
+		assertEquals("0", result.getHeader("content-length"));
+		assertEquals("*/*", result.getHeader("content-type"));
```

Comment:
I'm surprised that the content-type header is `*/*` and not JSON. Can you please set the mimetype on the `repeatableMessage` to JSON? Or otherwise the `produces` on the ApiListenerBuilder?